config/core/configmaps/gc.yaml

```diff
@@ -60,27 +60,52 @@ data:
     # V2 Garbage Collector Settings
     # ---------------------------------------
     #
-    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # These settings are enabled via the "responsive-revision-gc" feature flag.
     # ALPHA NOTE: This feature is still experimental and under active development.
+    #
+    # Stale
+    #   * Revisions are considered stale if they were created after more than
+    #      "retain-since-create-time" or last referenced by a route more than
+    #      "retain-since-last-active-time" ago.
+    #   * Either timeout may be set to "disabled" to turn off time-based GC.
+    # Active
+    #   * Revisions which are referenced by a Route are considered active and
+    #     are exempted from GC.
+    #   * Individual revisions may be marked with the annotation
+    #      "knative.dev/no-gc":"true" to be permanently considered active.
```

Comment:
I'd reorder to have `Active` first. Then `Stale` can be described as

1. not active
2. also time params.

Reply:
That better matches how the logic actually works too.